web/webhook.go

```diff
@@ -56,9 +55,13 @@ func incomingWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
-	if *c.App.Config().LogSettings.EnableWebhookDebugging {
-		mlog.Debug(fmt.Sprintf("Incoming webhook received. Id=%s Content=%s", id, incomingWebhookPayload.ToJson()))
-	}
+	defer func() {
```

Comment:
This defer should be at the top of the function. Otherwise it won't do anything. 
You are also checking the wrong error. We want to check if we are returning an error and log it.

Reply:
@crspeller Thanks very much for your feedback! I addressed your comments and updated the PR. Please let me know how it looks.